charts/datadog/values.yaml

```diff
@@ -706,6 +706,11 @@ datadog:
       # datadog.sbom.containerImage.enabled -- Enable SBOM collection for container images
       enabled: false
 
+      # datadog.sbom.containerImage.uncompressedLayersSupport -- Use container runtime snapshotter
+      # This should be set to true when using EKS, GKE or if containerd is configured to
+      # discard uncompressed layers.
```

Comment:
could you add in the comment that it will force the addition of the `SYS_ADMIN` capability in the agent container's security context even if the user has defined their own `securityContext`.

Reply:
Done